client/store/plugins/vuex-persist.js

```diff
@@ -0,0 +1,9 @@
+import VuexPersistence from 'vuex-persist';
+
+export default new VuexPersistence({
+  // The key to store the state in the storage provider.
+  key: process.env.VUEX_STORAGE_KEY,
+  modules: ['auth'],
+  // Use localStorage
+  storage: window.localStorage
```

Comment:
Same as above because likely library itself has some sort of in-code docs (typings, jsdoc) and editor will show them as needed. :bowtie: